cmake/libuv.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(gRPC_LIBUV_PROVIDER STREQUAL "module")
+  if(NOT LIBUV_ROOT_DIR)
+    set(LIBUV_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR}/third_party/libuv)
+  endif()
+  if(EXISTS "${LIBUV_ROOT_DIR}/CMakeLists.txt")
+    add_subdirectory(${LIBUV_ROOT_DIR} third_party/libuv)
+    if(TARGET uv_a)
```

Comment:
out of curiosity, why is the target called uv_a?

Reply:
If I understand it correctly, this target name must be the same as that created in the libuv CMakeLists.txt, which uses [uv_a](https://github.com/libuv/libuv/blob/v1.35.0/CMakeLists.txt#L271) for the static library.

Reply:
yeah, that sounds correct, just wondering why they are using such a weird name.